serializations.md

```diff
@@ -455,28 +456,41 @@ The channel offchain transaction is not included directly in the transaction tre
 * The channel slash transaction.
 
 ```
-[ <channel_id>       , binary()
-, <previous_round>   , int()
-, <round>            , int()
-, <initiator>        , binary()
-, <responder>        , binary()
-, <initiator_amount> , int()
-, <responder_amount> , int()
-, <updates>          , [{int(), binary(), binary(), int()}]
-, <state>            , binary()
+[ <channel_id>       :: binary()
+, <previous_round>   :: int()
+, <round>            :: int()
+, <initiator>        :: binary()
+, <responder>        :: binary()
+, <initiator_amount> :: int()
+, <responder_amount> :: int()
+, <updates>          :: [{int(), binary(), binary(), int()}]
+, <state>            :: binary()
 ]
 ```
 
 #### Channel
 ```
-[ <id>              , binary()
-, <initiator>       , binary()
-, <responder>       , binary()
-, <total_amount>    , int()
-, <initiator_amount>, int()
-, <channel_reserve> , int()
-, <round>           , int()
-, <lock_period>     , int()
-, <closes_at>       , int()
+[ <id>               :: binary()
+, <initiator>        :: binary()
+, <responder>        :: binary()
+, <total_amount>     :: int()
+, <initiator_amount> :: int()
+, <channel_reserve>  :: int()
+, <round>            :: int()
+, <lock_period>      :: int()
+, <closes_at>        :: int()
+]
+```
+
+#### Proof of inclusion on state trees (POI)
+```
+[ {<accounts>  :: [<proof_of_inclusion> :: {<root_hash> :: binary(), [{<mpt_hash> :: binary(), [<mpt_value> :: binary()]}]}]}
+, {<calls>     :: [<proof_of_inclusion> :: {<root_hash> :: binary(), [{<mpt_hash> :: binary(), [<mpt_value> :: binary()]}]}]}
+, {<channels>  :: [<proof_of_inclusion> :: {<root_hash> :: binary(), [{<mpt_hash> :: binary(), [<mpt_value> :: binary()]}]}]}
+, {<contracts> :: [<proof_of_inclusion> :: {<root_hash> :: binary(), [{<mpt_hash> :: binary(), [<mpt_value> :: binary()]}]}]}
+, {<ns>        :: [<proof_of_inclusion> :: {<root_hash> :: binary(), [{<mpt_hash> :: binary(), [<mpt_value> :: binary()]}]}]}
+, {<oracles>   :: [<proof_of_inclusion> :: {<root_hash> :: binary(), [{<mpt_hash> :: binary(), [<mpt_value> :: binary()]}]}]}
 ]
 ```
+NOTE: `[{<mpt_hash>, <mpt_value>}]` is the sorted list of Merkel Patricia Tree nodes in the proof. If the subtree is empty,
+the serialization is just `[]`, and otherwise it is a list of one element `[<proof_of_inclusion>]`.
```

Comment:
Ah I understood now:
* Each tree must be present;
* Each tree must have exactly one `<proof_of_inclusion>` item.

hence the root hash of each of the trees is always present hence the state hash of the corresponding block header can be computed.